examples/app/sliders_applet/sliders_app.py

```diff
@@ -85,10 +94,20 @@ def create(cls):
         return obj
 
     def setup_events(self):
+        """Attaches the on_change event to the value property of the widget.
+
+        The callback is set to the input_change method of this app.
+
+        :return: None
```

Comment:
small change: we use Google style docstrings consistently, that is what `sphinx-napoleon` expects. So this should just be:
```
    Returns:
        None
````
Here is a full reference: http://sphinxcontrib-napoleon.readthedocs.org/en/latest/example_google.html